fury/actors/peak.py

```diff
@@ -0,0 +1,343 @@
+                elif len(colors) == num_pnts:
+                    if colors.ndim == 1:  # Scalar per point
+                        color_array = numpy_support.numpy_to_vtk(colors,
+                                                                 deep=True)
+                        colors_are_scalars = True
+                    elif colors.ndim == 2:  # RGB(A) color per point
+                        global_opacity = 1 if colors.shape[1] == 3 else -1
+                        color_array = numpy_to_vtk_colors(255 * colors)
+        color_array.SetName('colors')
+        return color_array, colors_are_scalars, global_opacity
+
+    @staticmethod
+    def _points_to_vtk_cells(points):
+        """
+        Returns the VTK cell array for the peaks given the set of points
+        coordinates.
+
+        Parameters
+        ----------
+        points : (N, 3) array or ndarray
+            points coordinates array.
+
+        Returns
+        -------
+        cell_array : vtkCellArray
+            connectivity + offset information.
+
+        """
+        pnts_per_line = 2
+        num_pnts = len(points)
+        num_cells = num_pnts // pnts_per_line
+
+        cell_array = vtk.vtkCellArray()
+
+        if vtk.vtkVersion.GetVTKMajorVersion() >= 9:
+            """
+            Connectivity is an array that contains the indices of the points
+            that need to be connected in the visualization. The indices start
+            from 0.
+            """
+            connectivity = np.array(list(range(0, num_pnts)), dtype=int)
+            """
+            Offset is an array that contains the indices of the first point of
+            each line. The indices start from 0 and given the known geometry of
+            this actor the creation of this array requires a 2 points padding
+            between indices.
+            """
+            offset = np.array(list(range(0, num_pnts + 1, pnts_per_line)),
+                              dtype=int)
+
+            vtk_array_type = numpy_support.get_vtk_array_type(
+                connectivity.dtype)
+            cell_array.SetData(
+                numpy_support.numpy_to_vtk(offset, deep=True,
+                                           array_type=vtk_array_type),
+                numpy_support.numpy_to_vtk(connectivity, deep=True,
+                                           array_type=vtk_array_type))
+        else:
+            connectivity = []
+            i_pos = 0
+
+            while i_pos < num_pnts:
+                end_pos = i_pos + pnts_per_line
+                """
+                In old versions of VTK (<9.0) the connectivity array should
+                include the length of each line and immediately after the
+                indices of the points in each line.
+                """
+                connectivity += [pnts_per_line]
+                connectivity += list(range(i_pos, end_pos))
+                i_pos = end_pos
+
+            connectivity = np.array(connectivity)
```

Comment:
same comment as before, you can work directly with an array

Reply:
Changed to numpy array.